corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -52,7 +53,13 @@ def post(self, request, *args, **kwargs):
 
         # reset the session's expiry if there's some formplayer activity
         secure_session = session.get('secure_session')
-        timeout = settings.SECURE_TIMEOUT if secure_session else settings.INACTIVITY_TIMEOUT
+        secure_session_timeout = settings.SECURE_TIMEOUT
+        domain = data.get('domain')
+        if domain:
+            domain_obj = Domain.get_by_name(domain)
```

Comment:
we don't have `domain_obj` on the `request` object at this point?

Reply:
I wasn't 100% sure for this view, and it's cached anyway. Do you know what would enforce that? `UsersMiddleware` populates `request.domain` with the name, but I didn't find the place that populates the full object.

Reply:
Yeah, I'm not sure offhand, just seems like something I'd have expected to be available. But agree with caching it doesn't really matter much.